test/Tizen.NUI.WindowSystem.Samples/Tizen.NUI.WindowSystem.Samples.cs

```diff
@@ -0,0 +1,322 @@
+                Size = new Size(400, 100),
+                Position2D = new Position2D(50, 1040),
+                Margin = 10,
+            };
+            Button BtnShow = new Button()
+            {
+                Text = "Show",
+                Size = new Size(200, 100),
+                Position2D = new Position2D(470, 850),
+                Margin = 10,
+            };
+            Button BtnHide = new Button()
+            {
+                Text = "Hide",
+                Size = new Size(200, 100),
+                Position2D = new Position2D(470, 990),
+                Margin = 10,
+            };
+
+            BtnScrollableSet.ClickEvent += BtnScrollableSet_ClickEvent;
+            BtnScrollableUnset.ClickEvent += BtnScrollableUnset_ClickEvent;
+            BtnScrollableRetain.ClickEvent += BtnScrollableRetain_ClickEvent;
+            BtnShow.ClickEvent += BtnClientShow_ClickEvent;
+            BtnHide.ClickEvent += BtnClientHide_ClickEvent;
+
+            Window.Instance.GetDefaultLayer().Add(BtnScrollableSet);
+            Window.Instance.GetDefaultLayer().Add(BtnScrollableUnset);
+            Window.Instance.GetDefaultLayer().Add(BtnScrollableRetain);
+            Window.Instance.GetDefaultLayer().Add(BtnShow);
+            Window.Instance.GetDefaultLayer().Add(BtnHide);
+        }
+
+        private void BtnScrollableSet_ClickEvent(object sender, Button.ClickEventArgs e)
+        {
+            qpClient.SetScrollable(Shell.QuickPanelClient.StateScrollable.Set);
+            textClientScrollable.Text = $"Scrollable: {qpClient.GetScrollable()}";
+        }
+        private void BtnScrollableUnset_ClickEvent(object sender, Button.ClickEventArgs e)
+        {
+            qpClient.SetScrollable(Shell.QuickPanelClient.StateScrollable.Unset);
+            textClientScrollable.Text = $"Scrollable: {qpClient.GetScrollable()}";
+        }
+        private void BtnScrollableRetain_ClickEvent(object sender, Button.ClickEventArgs e)
+        {
+            qpClient.SetScrollable(Shell.QuickPanelClient.StateScrollable.Retain);
+            textClientScrollable.Text = $"Scrollable: {qpClient.GetScrollable()}";
+        }
+        private void BtnClientShow_ClickEvent(object sender, Button.ClickEventArgs e)
+        {
+            qpClient.Show();
+        }
+        private void BtnClientHide_ClickEvent(object sender, Button.ClickEventArgs e)
+        {
+            qpClient.Hide();
+        }
+
+        private void BtnService_ClickEvent(object sender, Button.ClickEventArgs e)
+        {
+            Window.Instance.GetDefaultLayer().Remove(BtnService);
+            Window.Instance.GetDefaultLayer().Remove(BtnClient);
+            qpService = new Shell.QuickPanelService(tzShell, Window.Instance, Shell.QuickPanelService.QuickPanelType.SystemDefault);
+
+            TextLabel textServiceType = new TextLabel($"Type: {qpService.GetType()}");
+            textServiceType.Position2D = new Position2D(0, -300);
```

Comment:
Using Position(0, -300) is recommended instead of Position2D.

Reply:
I changed all Position2D to Position in this code